tests/testthat/test-create_project.R

```diff
@@ -0,0 +1,53 @@
+
+test_that("create_project() basic functionality test", {
```

Comment:
Yep this has been done. I've used `suppressMessages()` around the `create_project()` call in test script.

I wonder though if you would also be against this nosiness when a user uses the function to create a project, as it will output those messages (due to this being what the `{usethis}` functions like `usethis::use_testthat()` outputs).